src/pages/RFOX/hooks/useEarliestBlockNumberByTimestampQuery/test-data/blocks.ts

```diff
@@ -0,0 +1,4651 @@
+import type { Block } from 'viem'
+
+export const blocks: Block[] = [
```

Comment:
tons of mock data for only needing block.height and block.timestamp haha. could just cast as Block[], knowing it fulfills the data reqs and save 4.5k extra lines in the codebase as an option

Reply:
great idea!
151404b4b595d2a8b7ed17a165868dbc0e0dcd37